README.md

```diff
@@ -150,25 +150,20 @@ With the Postgres container enabled, users (both students and instructors) can c
 
 ### Spawn Specific Workspace Types
 
-> **New in Version 0.8.0**: users can spawn specific workspace types by setting the `workspace_type` key when initiating the launch request. The `workspace_type` key can either be added as a query parameter or included within the request's body (when initiating an LTI launch).
-
-To enable this feature, you must set the `JupyterHub.spawner_class` to the `IllumiDeskWorkSpaceDockerSpawner` class. Like the `IllumiDeskRoleDockerSpawner` class, the `IllumiDeskWorkSpaceDockerSpawner` class inherits from the `IllumiDeskBaseDockerSpawner` class which manages the logic to assign the image to spawn based on the `workspace_type` key in the `auth_state` dictionary.
-
 Additional workspace types include:
 
 - THEIA IDE: `theia`
 - RStudio: `rstudio`
 - VS Code: `vscode`
 
```

Comment:
Let's remove this bullet list and add:

Additional workspace types are supported by any workspace type that is supported by the underlying [`jupyter-server-proxy`] package. This stack has been tested with a variety of workspace types including:

**IDEs**

- Theia
- RStudio
- VS Code

**Data Tools**

- OpenRefine

**Visualization/Dashboard Servers**

- Plotly Dash
- Streamlit
- Bokeh Server
- Jupyter Voila